src/sage/combinat/recognizable_series.py

```diff
@@ -1913,6 +1931,59 @@ def some_elements(self):
                     break
                 yield self(mu, *LR)
 
+    @cached_method
+    def _zero_(self):
+        r"""
+        Return the zero element of this :class:`RecognizableSeriesSpace`,
+        i.e. the unique neutral element for `+`.
+
+        TESTS::
+
+            sage: Rec = RecognizableSeriesSpace(ZZ, [0, 1])
+            sage: Z = Rec._zero_(); Z
+            0
+            sage: Z.linear_representation()
+            ((), Finite family {0: [], 1: []}, ())
+        """
+        from sage.matrix.constructor import Matrix
+        from sage.modules.free_module_element import vector
+        from sage.sets.family import Family
+
+        return self.element_class(
+            self, Family(self.alphabet(), lambda a: Matrix()),
+            vector([]), vector([]))
+
+    @cached_method
+    def one(self):
+        r"""
+        Return the one element of this :class:`RecognizableSeriesSpace`,
+        i.e. the embedding of the unique neutral element for `*` in the
+        coefficient ring.
```

Comment:
I do not yet understand how `*` is defined in the space of recognizable series.

Reply:
perhaps
"embedding of the one of the coefficient ring into this  :class:`RecognizableSeriesSpace`."

Reply:
4ed1e49e790

Reply:
Thank you, please resolve.